README.rst

```diff
@@ -94,6 +123,38 @@ Contributing
 
 See https://docs.scrapy.org/en/master/contributing.html for details.
 
+If you are a new contributor, you may benefit from starting at the following locations. This will provide you
+a place to begin your contribution journey:
+
+* `Overview of Scrapy`_
+* `Writing New Patches`_
+* `Submitting Patches`_
+* `Reporting Bugs`_
+* `Documentation Policies`_
+
+.. _Overview of Scrapy: https://docs.scrapy.org/en/master/intro/overview.html
+.. _Writing New Patches: https://docs.scrapy.org/en/master/contributing.html#writing-patches
+.. _Reporting Bugs: https://docs.scrapy.org/en/master/contributing.html#reporting-bugs
+.. _Submitting Patches: https://docs.scrapy.org/en/master/contributing.html#submitting-patches
+.. _Coding Style: https://docs.scrapy.org/en/master/contributing.html#coding-style
+.. _Documentation Policies: https://docs.scrapy.org/en/master/contributing.html#documentation-policies
+
+Also, join our social media communities and get acquainted with the developer community! This will help you learn and
```

Comment:
This duplicates https://scrapy.org/community/ already linked in the Community section.